virt/virtual_machines/virt-accessing-vm-ssh.adoc

```diff
@@ -52,7 +52,10 @@ You can also add the key to a project by using the {product-title} web console.
 :static-key:
 include::modules/virt-adding-key-creating-vm-template.adoc[leveloffset=+3]
 
+// Hiding in ROSA/Dedicated as this is a Dev Preview feature
+ifndef::openshift-rosa,openshift-dedicated[]
 include::modules/virt-creating-vm-instancetype.adoc[leveloffset=+3]
+endif::openshift-rosa,openshift-dedicated[]
```

Comment:
Same question here re: instance types support. If they are supported in 4.15+, should this be hidden?